addons/website/views/snippets.xml

```diff
@@ -1147,7 +1151,8 @@
 
     <!-- Carousel | .s_carousel | .s_quotes_carousel -->
     <div data-js="carousel"
-        data-selector=":not(.o_gallery > .container) > .carousel">
+        data-selector=".s_carousel_wrapper, .s_quotes_carousel_wrapper"
```

Comment:
Add another class on both snippet `o_carousel_wrapper` and only use that one here. That way when we will make other carousel snippets we won't have to change something here